pyiceberg/manifest.py

```diff
@@ -262,15 +346,13 @@ class DataFile(Record):
         "split_offsets",
         "equality_ids",
         "sort_order_id",
-        "spec_id",
     )
     content: DataFileContent
     file_path: str
     file_format: FileFormat
     partition: Record
     record_count: int
     file_size_in_bytes: int
-    block_size_in_bytes: Optional[int]
```

Comment:
I've removed this since we shouldn't use it. It is still part of the schema itself.